pages/backgrounds.fr.md

```diff
@@ -0,0 +1,14 @@
+<!--
+.. link:
+.. description:
+.. tags: Backgrounds
+.. date: 2014-02-24 17:32:07
+.. title: Backgrounds
```

Comment:
You can also translate page title, otherwise the title will be displayed as `Backgrounds` on page content and browser titlebar.

Reply:
Done, thanks.